### PR TITLE
link fix

### PR DESCRIPTION
**Pull request by chomipi88**

### CODE REVIEW

File: astro/release-notes.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ You can now [create new Clusters](create-cluster.md) in:
 - `ap-northeast-3` (Osaka)  
 - `me-south-1` (Bahrain)
 
-For a full list of AWS regions supported on Astro, see [AWS Resource Reference](https://docs.astronomer.io/resource-reference-aws.md#aws-region).
+For a full list of AWS regions supported on Astro, see [AWS Resource Reference](resource-reference-aws#aws-region).
```
Comment (jwitz) [inline astro/release-notes.md:58]:
```suggestion
For a full list of AWS regions supported on Astro, see [AWS Resource Reference](resource-reference-aws.md#aws-region).
```

Comment (paolaperaza) [inline astro/release-notes.md]:
```suggestion
For a full list of AWS regions supported on Astro, see [AWS Resource Reference](resource-reference-aws.md#aws-region).
```

**Review by jwitz — APPROVED** [review]
LGTM, thanks for the catch!

**Review by paolaperaza — APPROVED** [review]
✅ ✅ ✅

**Comment by paolaperaza** [comment]
Lol @jwitz literal jinx!

**Pull request merged**